db/migrate/20230724205840_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
```

Comment:
- [ ] The `Post` table should contain a `:likes_counter` and `:comments_counter` with the type of integer, kindly make sure to implement this